site/cypress/integration/ui_tests/e2e_tests/01-site-navigation.spec.js

```diff
@@ -61,6 +61,16 @@ describe("Navigating the site and reading articles", function() {
         })
     });
 
+    it('An error in an article without roles', function () {
+      cy.visit(globalTestData.deepArticlePath);
+      cy.get(article.roles).should('have.text', 'Error please notify Field Staff Support there are no roles associated with this article');
```

Comment:
Would be nice if we could test for the colour of this text being visually distinct, if that's possible.  It's supposed to be red, but that could change I suppose so a less brittle test might be to check it's not the same as some vanilla text on the same page?

Reply:
I've added a test for red text